vespa-http-client/src/main/java/com/yahoo/vespa/http/client/core/communication/ClusterConnection.java

```diff
@@ -102,6 +102,13 @@ public ClusterConnection(OperationProcessor operationProcessor,
         }
     }
 
+    private static ObjectMapper createMapper() {
```

Comment:
Duplicated 3 times. A candidate for vespajlib (it already depends on a few jackson artifacts)?